toolchain/check/check.cpp

```diff
@@ -320,9 +320,12 @@ auto CheckParseTrees(llvm::MutableArrayRef<Unit> units, bool prelude_import,
   // UnitAndImports is big due to its SmallVectors, so we default to 0 on the
   // stack.
   llvm::SmallVector<UnitAndImports, 0> unit_infos;
+  llvm::SmallVector<Parse::GetTreeAndSubtreesFn> all_trees_and_subtrees;
```

Comment:
WDYT about naming this `all_tree_getters` or something to indicate these are callable things, not the trees themselves?

(And maybe can just mention `tree` here given its just a local variable?)

Reply:
Renamed to `tree_and_subtree_getters`. I prefer to include the "and subtree" given we typically say "tree" for `Parse::Tree`. I'm assuming you suggested "all" because I had it before, but I was just trying to emphasize the plural there.